tests/test_parsl_flux_mpi_hello.py

```diff
@@ -136,6 +136,11 @@ def test_run_mpi_hello(load_config):
         for line in f:
             assert re.match(r'Hello world from host \S+, rank \d+ out of 6', line)
 
+    with open('parsl_flux_mpi_hello_run.out', 'r') as f:
+        for line in f:
+            assert re.match(r'Hello world from host \S+, rank \d+ out of 6', line)
+
+
```

Comment:
Interesting, after I did a `git pull` and merged main into my bugfix branch, I did a `git commit` and pushed here and this showed up.

Reply:
I'll discard these lines since they seem to be replicated from the above ones